api/src/main/java/io/druid/data/input/InputRow.java

```diff
@@ -28,8 +30,8 @@
  * implement "schema-less" data ingestion that allows the system to add new dimensions as they appear.
  *
  */
-public interface
-    InputRow extends Row
+@PublicApi
```

Comment:
Hmm, thanks for looking into that. In that case I think it makes sense to make this an extension point. I'll change it.